src/BulletSoftBody/btSoftBody.h

```diff
@@ -163,14 +163,17 @@ class btSoftBody : public btCollisionObject
 			RVSmask = 0x000f,  ///Rigid versus soft mask
 			SDF_RS = 0x0001,   ///SDF based rigid vs soft
 			CL_RS = 0x0002,    ///Cluster vs convex rigid vs soft
-            SDF_RD = 0x0003,   ///DF based rigid vs deformable
-            SDF_RDF = 0x0004,   ///DF based rigid vs deformable faces
+			SDF_RD = 0x0004,   ///SDF based rigid vs deformable
 
-			SVSmask = 0x00F0,  ///Rigid versus soft mask
-			VF_SS = 0x0010,    ///Vertex vs face soft vs soft handling
```

Comment:
with this change, VF_SS (0x010 -> 0x100) has the same mask as SDF_RDF (0x100), is that intentional

Reply:
Done.